security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -0,0 +1,77 @@
++
+* In this release, the remediation template now allows multi-value variables. 
++
+* With this update, the Compliance Operator can change remediations based on variables that are set in the compliance profile. This is useful for remediations that include deployment-specific values such as time outs, NTP server host names, or similar. Additionally, the `ComplianceCheckResult` objects now use the label `compliance.openshift.io/check-has-value` that lists the variables a check can use.
++
+* This enhancement removes the requirement to extend an existing Profile in order to create a Tailored Profile. The `extends` field in the `TailoredProfile` CRD is no longer mandatory. You can now select a list of rule objects to create a tailor profile from scratch. Note: You will need to select whether your profile applies to nodes or the platform by setting the `compliance.openshift.io/product-type:` annotation or by setting the `-node` suffix for the `TailoredProfile` CR.
++
+* Previously, only master node taints were tolerated. Now, the Compliance Operator tolerates all taints, making it possible to schedule scans on all nodes.
+
+
+[id="openshift-compliance-operator-0-1-44-bug-fixes"]
+=== Bug fixes
+
+* Previously, while performing a scan, a crash was occurring in one of the scanner containers of the pods performing the scan. In this release, the Compliance Operator uses the latest OpenSCAP version 1.3.5 to avoid this issue.
++
+* Previously, only one global `KubeletConfig` object was created per pool. In this release, a `KubeletConfig` object is created for each pool. 
++
+* The rules for tailored profiles are now validated so that only rules of the same type (Platform or Node) are included in the validation.
++
+* Remediations now have a `NeedsReview` state if they require a variable to be set. In order to automatically apply remediations, you must set all required variables before the MachineConfig pool is unpaused and the remediations applied.
++
+* Previously, the variable that was set for value-required handling was read from the datastream. Now, the variable is read directly from the tailoring to ensure that the appropriate remediation is created. 
++
+* The RBAC Role and RoleBinding used for Prometheus metrics is changed to `cluster-wide` to ensure that monitoring works out-of-the-box when using ClusterRole or ClusterRoleBinding for monitoring permissions.
++
+* Previously, if an error occurred while parsing a profile, rules or variables objects would be removed and deleted from the profile. Now, if an error occurs during parsing, the `profileparser` annotates the object with a temporary annotation that prevents the object from being deleted until after parsing completes. link:https://bugzilla.redhat.com/show_bug.cgi?id=1988259[(BZ#1988259)].
++
+* Because the XCCDF standard discourages empty titles and descriptions for tailored profiles, titles and descriptions are now required to be set in `TailoredProfile` CRs.
```

Comment:
Did the user see an error - is that what is meant by discourages?

Reply:
I don't remember to be honest. @JAORMX do you?

Reply:
@kmccarron-rh Yes, User see an error when they create tailored profiles with titles and descriptions.

Reply:
Thank you @pdhamdhe - I made a slight adjustment to the language.